tools/test_value_scraper.py

```diff
@@ -57,9 +57,9 @@ def do_it(*, config_file):
     hosts = si.content.rootFolder.childEntity[0].hostFolder.childEntity[0].host
     for host in hosts:
         config["esxi_host_name"] = host.name
-        config["esxi_datastore_disk_names"] = [
-            extent.diskName for datastore in host.datastore for extent in datastore.info.vmfs.extent
-        ]
+#        config["esxi_datastore_disk_names"] = [
+#            extent.diskName for datastore in host.datastore for extent in datastore.info.vmfs.extent
+#        ]
```

Comment:
**suggestion** probably comment this back in :upside_down_face:

Reply:
Done